### PR TITLE
Fixes being able to instantly create dense grilles

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: 
fix: Fixes being able to instantly create dense grilles
/:cl:

### CODE REVIEW

File: code/game/objects/structures/grille.dm [hunk 1]
```diff
@@ -200,7 +200,7 @@
 
 /obj/structure/grille/attackby(obj/item/W, mob/user, params)
 	user.changeNext_move(CLICK_CD_MELEE)
-	if(istype(W, /obj/item/stack/rods) && broken)
+	if(istype(W, /obj/item/stack/rods) && broken && do_after(user,20, target = src))
```
Comment (dragomagol) [inline code/game/objects/structures/grille.dm:203]:
```suggestion
	if(istype(W, /obj/item/stack/rods) && broken && do_after(user, 2 SECONDS, target = src))
```

**Comment by NamelessFairy** [comment]
Really massive slowdown for building windows. If the issue is creating hard stun traps could we just make it so that they don't get electrified until after a delay?

**Review by Fikou — CHANGES_REQUESTED** [review]
needs to take as much time (or less, maybe you can put that behind tinkerer gloves) as building them, not more

**Comment by Fikou** [comment]
> Really massive slowdown for building windows

why are you building windows off broken grilles

**Comment by NamelessFairy** [comment]
> > Really massive slowdown for building windows
> 
> why are you building windows off broken grilles

Nevermind was under the impression this was nerfing placing them in the first place after seeing the video earlier

**Comment by NamelessFairy** [comment]
Anyways now that that mixup is cleared up, gonna suggest reducing it down to a second so its the same as actually building them from scratch.

**Comment by ATH1909** [comment]
"fixes"

**Comment by improvedname** [comment]
> the title of this PR is very misleading; this actually applies to repairing existing, broken grilles. why is repairing them being instant a problem?

**Comment by CRITAWAKETS** [comment]
why two entire seconds

**Comment by moocowswag** [comment]
This is a fix?

**Comment by improvedname** [comment]
skillbased exploirs fr

**Comment by improvedname** [comment]
d

> This is a fix?
> 
> so anyone that was using instant grilles was exploiting?

not a bug, oversight.
bugs are like being able to infinitely stack ka cooldowns

**Pull request merged**